index.html

```diff
@@ -141,80 +141,3 @@ <h2>$1200</h2>
 </body>
 </html>
 
-<!--Following code has been replaced with FlexBOX
-ORIGINAL
-<section id="pricing">
-    <div class="product" id="tenor">
-        <div class="level">Tenor Trombone</div>
-        <h2>$600</h2>
-        <ol>
-            <li>Lorem ipsum.</li>
-            <li>Lorem ipsum.</li>
-            <li>Lorem ipsum dollar.</li>
-            <li>Lorem ipsum.</li>
-        </ol>
-        <button class="btn">Select</button>
-    </div>
-    <div class="product" id="bass">
-        <div class="level">Base Trombone</div>
-        <h2>$900</h2>
-        <ol>
-            <li>Lorem ipsum.</li>
-            <li>Lorem ipsum.</li>
-            <li>Lorem ipsum dollar.</li>
-            <li>Lorem ipsum.</li>
-        </ol>
-        <button class="btn">Select</button>
-    </div>
-    <div class="product" id="valve">
-        <div class="level">Valve Trombone</div>
-        <h2>$1200</h2>
-        <ol>
-            <li>Plays similar to a Trumpet</li>
-            <li>Great for Juzz Band</li>
-            <li>Lorem ipsum dollar.</li>
-            <li>Lorem ipsum.</li>
-        </ol>
-        <button class="btn">Select</button>
-    </div>
-</section>
-
-
-REPLACED WITH FLEX BOX
-<div class = "fcontainer">
-    <div class="product" id="tenor">
-        <div class="level">Tenor Trombone</div>
-        <h2>$600</h2>
-        <ol>
-            <li>Lorem ipsum.</li>
-            <li>Lorem ipsum.</li>
-            <li>Lorem ipsum dollar.</li>
-            <li>Lorem ipsum.</li>
```

Comment:
Nice work so far implementing the list 👌, But it will look better adding realistic data. _For _example, guitar, saxophone and other many.__